tests/x11regressions/empathy/empathy_irc.pm

```diff
@@ -48,16 +48,13 @@ sub run() {
     };
 
     # join openqa channel
-    if (sle_version_at_least('12-SP2')) {
+    send_key "ctrl-j";
+    if (!check_screen 'empathy-join-room', 4) {
```

Comment:
I like this `wait_screen_change { send_key 'ctrl-j' };` with `check_screen` `,0`. I'll use that. Thanks!